feelslikemonday/app/src/main/java/com/example/feelslikemonday/DAO/UserDAO.java

```diff
@@ -101,7 +103,9 @@ public void onComplete(@NonNull Task<DocumentSnapshot> task) {
                                 }
                                 return;
                             }
+
                             // Assumes only one document will be returned
+                             User curent = doc.toObject(User.class);
```

Comment:
I don't think we need this for now. If trying to debug, we can directly place `doc.toObject(User.class)` into our watchlist to get the value of the user